src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import ProfileMenu from '../ProfileMenu';
+import SideMenu from '../SideMenu/SideMenu.component';
+import { MenuIcon, SearchIcon, ProfileIcon } from '../../utils/icons';
+
+import '../../utils/styles/generalStyles.css';
+import './Header.styles.css';
+
+const HeaderContainer = styled.header`
+  background-color: #3234b8;
+  -webkit-box-shadow: 0px 10px 13px -7px #000000, 1px 1px 8px 1px rgba(0, 0, 0, 0);
+  box-shadow: 0px 10px 13px -7px #000000, 1px 1px 8px 1px rgba(0, 0, 0, 0);
+  color: white;
+  height: 3.5em;
+  padding: 0 1em;
+`;
+
+const HeaderElement = styled.div`
+  height: 2em;
+  margin: 0 10px;
+`;
```

Comment:
This could be moved to another part by creating 2 new components: `HeaderContainer`, `HeaderElement`, inside their own folders. This way the structure of the code is leaner and clearer.